content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -152,7 +160,7 @@ To view completed Pods of a Job, use `kubectl get pods`.
 To list all the Pods that belong to a Job in a machine readable form, you can use a command like this:
 
 ```shell
-pods=$(kubectl get pods --selector=job-name=pi --output=jsonpath='{.items[*].metadata.name}')
+pods=$(kubectl get pods --selector=batch.kubernetes.io/job-name=pi --output=jsonpath='{.items[*].metadata.name}')
```

Comment:
This reminds me.... how does `kubectl logs` work for job? If it uses the selector, it might not need to be updated.

Reply:
I don't think `kubectl logs` work on the job object.  I think you can only use `kubectl logs` on pods but I would love to be proven wrong.

Reply:
`kubectl logs job/{job-name}` :)

Reply:
Today I learned!  I added that to the doc as I wasn't aware that you could do that.

Reply:
> This reminds me.... how does `kubectl logs` work for job? If it uses the selector, it might not need to be updated.

I does use a selector, see https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/polymorphichelpers/helpers.go#L174 so that's one of the reasons we will need to support both for quite a while :sweat:

Reply:
Rather not!

Because it uses the selector, we don't need to keep adding the legacy labels not to break kubectl.

Of course, this doesn't apply to out-of-tree controllers using the labels directly.